src/controllers/TripController.js

```diff
@@ -0,0 +1,70 @@
+import dotenv from 'dotenv';
+import Customize from '../helpers/Customize';
+import emailHelper from '../helpers/EmailHelper';
+import { tripRequest, tripRequestCities } from '../database/models';
+import DataEngine from '../middlewares/DataEngine';
+
+
+dotenv.config();
+/**
+ * @export
+ * @class TripController
+ */
+class TripController {
+  /**
+   * User can be able to request multi-city trip
+   * @static
+   * @param {object} req request object
+   * @param {object} res response object
+   * @memberof TripController
+   * @returns {object} data
+   */
+  static async requestTrip(req, res) {
+    try {
+      const userId = req.user.id;
+      const {
+        reason, city, startDate, returnDate
+      } = req.body;
+      const locationArr = city.split(',');
```

Comment:
Please how are you sending the cities that needs this spliting. You should receive it as an array or cities.

Reply:
Thanks for the feedback, I have implemented it.